src/photos.cs

```diff
@@ -1162,22 +1167,35 @@ interface PHPhotoLibrary {
 		[TV (13,0), Mac (10,15), iOS (13,0)]
 		[Export ("unregisterAvailabilityObserver:")]
 		void Unregister (IPHPhotoLibraryAvailabilityObserver observer);
+
+		[TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Field ("PHLocalIdentifiersErrorKey")]
+		NSString PHLocalIdentifiersErrorKey { get; }
```

Comment:
not sure if it's the best place for that key
but it's not named correctly, IOW you do not want to use `PHPhotoLibrary.PHLocalIdentifiersErrorKey`
minimally remove `PH` prefix
also check if other constants exists - there might be a better place for this one (or not)

Reply:
WE already did that :) That key is only for when the indentifer is not matched and we noticed that the only Object with methods for identifiers is this class, so it would be something like:
```csharp
PHPhotoLibrary.LocalIdentifierMappingsForCloudIdentifiers(.., out var error);
// access the error info
error.UserInfo[PHPhotoLibrary.LocalIdentifiersErrorKey]
```

Yet, I did not know that @rachelkang was going to add those two methods as a Category in a diff object, I would have added them in the library and not in PHPhotoLibrary_CloudIdentifiers. If we want a diff class for the category, lets move the key there, else lets move the category methods to  PHPhotoLibrary. My preference is  PHPhotoLibrary but I'm happy either way.

Reply:
sounds good - but the
> remove `PH` prefix
remains :)

Reply:
removed the prefix!

Let me know if I should definitively move the key to PHPhotoLibrary_CloudIdentifiers or move the category methods to PHPhotoLibrary. Moving neither for now, although I think it would make sense to move the key to PHPhotoLibrary_CloudIdentifiers where the category methods are. I put the category methods there because the older versions of them (deprecated) also live there